powersimdata/design/investment/investment_costs.py

```diff
@@ -50,39 +48,29 @@ def calculate_ac_inv_costs(
     scenario,
     sum_results=True,
     exclude_branches=None,
-    base_grid=None,
 ):
     """Calculate cost of upgrading AC lines and/or transformers in a scenario.
     NEEM regions are used to find regional multipliers.
 
     :param powersimdata.scenario.scenario.Scenario scenario: scenario instance.
     :param bool sum_results: whether to sum data frame for each branch type. Defaults to
         True.
-    :param powersimdata.input.grid.Grid base_grid: a Grid to compare against. If None,
-        the grid model and interconnect from the ``scenario`` are used to instantiate a
-        corresponding unmodified Grid.
     :return: (*dict*) -- keys are {'line_cost', 'transformer_cost'}, values are either
         float if ``sum_results``, or pandas Series indexed by branch ID.
         Whether summed or not, values are $USD, inflation-adjusted to today.
     """
     grid = scenario.state.get_grid()
-    if base_grid is None:
-        base_grid = Grid(
-            scenario.info["interconnect"].split("_"), source=scenario.info["grid_model"]
-        )
-    else:
-        _check_grid_models_match(base_grid, grid)
-
-    grid_new = cp.deepcopy(grid)
```

Comment:
Note, I removed this deepcopy because looking at the `get_grid` functions, there's already a deepcopy being made, so this call seemed useless, correct me if I'm wrong though

Reply:
I agree. Then we don't need l.64: `grid = cp.deepcopy(grid)`, do we?

Reply:
good catch. done